lib/formatters/ruby.js

```diff
@@ -18,12 +18,16 @@ module.exports = function formatRuby(code, filepath) {
 
   if (configPath !== null) {
     configOption = `--config ${configPath}`;
+  } else {
+    configOption = "";
   }
 
   fs.writeFileSync(tmpobj.name, code);
   try {
+    let shellCommand = `rubocop -f j ${configOption} --auto-correct --stdin ${tmpobj.name}`
```

Comment:
Love this.  Maybe a const? this value should not change in the scope of this function.